script/DeployToken.s.sol

```diff
@@ -5,17 +5,30 @@ import {Script, console} from "forge-std/Script.sol";
 import {TokenFactory, InstantLiquidityToken} from "../src/TokenFactory.sol";
 
 contract DeployToken is Script {
-
     function run() public {
         vm.broadcast();
         _run(0x5eD5A9b33019a2C08AAD51dD8ebbB75acA70c6A1);
     }
-    function _run(address _factory) public returns (InstantLiquidityToken) {
+
+    function _run(address _factory) public returns (InstantLiquidityToken, uint256) {
+        TokenFactory factory = TokenFactory(_factory);
+        (InstantLiquidityToken token, uint256 lpTokenId) = factory.deploy("", "");
```

Comment:
the `deploy` args don't matter here because the metal CLI is going to replace them right?

Reply:
because we want the fields to be blank when the user opens the configure page